src/Collapsible/index.js

```diff
@@ -72,6 +72,7 @@ class Collapsible extends Component {
               style: {
                 marginBottom: 0,
               },
+              className: `${baseClass}__clickable-node`,
```

Comment:
I've always disliked the prop name `clickableNode`, even though it's pretty semantic. Any other suggestions?

Reply:
@jacobsfletch  what about something with the word `Trigger` in it